src/components/sbb-link/sbb-link.tsx

```diff
@@ -63,33 +71,107 @@ export class SbbLink {
   @Prop() public textSize: InterfaceLinkAttributes['textSize'] = 's';
 
   /**
-   * Choose the link style variant.
+   * Applies link inline styles (underline, inherit coloring/font-size etc).
    */
   @Prop() public variant: InterfaceLinkAttributes['variant'] = 'block';
 
-  private get _inlineVariant(): boolean {
-    return this.variant === 'inline' || this.variant === 'inline-negative';
-  }
+  /**
+   * Disabled attribute if link is used as button (optional)
+   */
+  @Prop() public disabled?: boolean;
 
-  public render(): JSX.Element {
-    const textSizeClass = this._inlineVariant ? '' : ` sbb-link--text-${this.textSize}`;
+  /**
+   * Name attribute if link is used as button (optional)
+   */
+  @Prop() public name?: string;
+
+  /**
+   * Form attribute if link is used as button (optional)
+   */
+  @Prop() public form?: string;
+
+  /**
+   * Type attribute if link is used as button (optional)
+   */
+  @Prop() public type?: InterfaceLinkAttributes['buttonType'];
+
+  /** This will be forwarded as aria-describedby to the relevant nested element. */
+  @Prop() public accessibilityDescribedby?: string;
+
+  /** This will be forwarded as aria-label to the relevant nested element. */
+  @Prop() public accessibilityLabel?: string;
+
+  /** This will be forwarded as aria-labelledby to the relevant nested element. */
+  @Prop() public accessibilityLabelledby?: string;
```

Comment:
Changes due to the AccessibilityProperties interface. Please rebase your PR if you don't find AccessibilityProperties interface.
```suggestion
  /** This will be forwarded as aria-label to the relevant nested element. */
  @Prop() public accessibilityLabel: string | undefined;

  /** This will be forwarded as aria-describedby to the relevant nested element. */
  @Prop() public accessibilityDescribedby: string | undefined;

  /** This will be forwarded as aria-labelledby to the relevant nested element. */
  @Prop() public accessibilityLabelledby: string | undefined;
```
```